photon-lib/src/main/scala/com/linkedin/photon/ml/hyperparameter/tuner/AtlasTuner.scala

```diff
@@ -0,0 +1,63 @@
+/*
```

Comment:
So this will now move out of photon-ml into atlas-ml by depending on photon-ml by the trait correct?

Reply:
Yes